qldbdriver/result.go

```diff
@@ -108,3 +154,39 @@ func (result *BufferedResult) Next() bool {
 func (result *BufferedResult) GetCurrentData() []byte {
 	return result.ionBinary
 }
+
+// GetConsumedIOs returns the statement statistics for the total number of read IO requests.
```

Comment:
In the comment, shall we use `usage` rather than `requests` to make it consistent with the API name?

Reply:
To be consistent with the [SDK docstring](https://docs.aws.amazon.com/sdk-for-go/api/service/qldbsession/#ExecuteStatementResult), I've updated to: 

`// GetConsumedIOs returns the statement statistics for the total number of read IO requests that were consumed.`